wp-content/themes/core/assets/js/admin.js

```diff
@@ -1 +1,13 @@
+/**
+ * unregister some block styles we don't need
+ */
+
+wp.domReady( () => {
+	if ( wp.blocks ) {
+		wp.blocks.unregisterBlockStyle( 'core/separator', 'default' );
+		wp.blocks.unregisterBlockStyle( 'core/separator', 'wide' );
+		wp.blocks.unregisterBlockStyle( 'core/separator', 'dots' );
+	}
+} );
```

Comment:
We haven't talked much about the JS directory's architecture. I reviewed SQ1 and felt like it was a lot that we potentially don't need. I figured more discussion and a separate PR could be made for building it out, which is why this is implemented in a super basic way here.